### ISSUE TITLE
Export logs to file

### ISSUE THREAD

**Issue by rappen** [rappen/PluginTraceViewer#11]
Export for further analysis in Excel or similar would be great.
Option to include message and exception.

**Comment by seanmcne** [comment]
👍 Would love to see this

**Comment by rappen** [comment]
There is an option already doing this. Saves as xml, perhaps not the easies of formats...

![image](https://cloud.githubusercontent.com/assets/2572253/25564637/e3c14bb4-2db7-11e7-9072-39242b7563dc.png)

**Comment by seanmcne** [comment]
Yeah my thought was for quick filtering and searching (quick & optionally complex) you really can't beat the excel format. 

Actually, I think excel has pretty good xml support now.  After doing some work in Bing I found an older xslt that you could try using to quickly transform the xml doc and dump it to an excel xml file? 

http://www.techrepublic.com/article/convert-an-xml-file-into-an-excel-spreadsheet-with-this-technique/ 

Here's another example - it's not 100% a slam dunk but it's pretty close? 

https://forums.asp.net/t/1520473.aspx?Xslt+to+convert+XML+file+to+Excel

**Comment by rappen** [comment]
Am I late? 😂

Did you try to export logs (Save - Logs... in the menu) and open that XML file in Excel?
A bit blurry, but the data is there...

**Comment by rappen** [comment]
SOLVED, less than a decade after the request! 🎉
See details in #18

Ping @seanmcne
